# Array of waypoints for global path
std_msgs/Header header
Wpnt[] waypoints
